dbms/src/DataStreams/tests/CMakeLists.txt

```diff
@@ -13,14 +13,3 @@
 # limitations under the License.
 
 include_directories (${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
this useless CMakeLists.txt can also be removed

Reply:
there is a gtest left, can not removed.